Geometry/MTDCommonData/data/etl/v7/etl.xml

```diff
@@ -0,0 +1,4021 @@
+    <Constant name="glueETROC_translation_z" value="1.185*mm"/>
+    <Constant name="modulePCB_translation_z" value="0.935*mm"/>
+    <Constant name="connectorsGap_translation_z" value="0.36*mm"/>
+    <Constant name="readoutBoard_translation_z" value="-0.49*mm"/>
+    <Constant name="servicesSensorModule_translation_z" value="-2.39*mm"/>
+    <Constant name="EModule_Timingactive_translation_z" value="-0.125*mm"/>
+    <Constant name="LGAD_Substrate_translation_z" value="0.025*mm"/>
+    <Constant name="PowerBoard_translation_z" value="1*mm"/>
+    <Constant name="servicesServiceHybrid_translation_z" value="-2*mm"/>
+    <Constant name="SensorModule_translation_z" value="0.21*mm"/>
+    <Constant name="ServiceHybrid_translation_z" value="1.*mm"/>
+    <Constant name="xoffset_servicehybrid" value="1*mm"/>
+
+    <Vector name="StartCopyNo_Front_Right" type="numeric" nEntries="28">
+      1, 7, 18, 33, 50, 69, 90, 112, 136, 161, 186, 207, 227, 247, 266,
+      285, 305, 325, 349, 374, 398, 421, 443, 463, 481, 497, 510, 517
+    </Vector>
+
+    <Vector name="StartCopyNo_Front_Left" type="numeric" nEntries="28">
+      1, 8, 21, 37, 55, 75, 97, 120, 144, 169, 193, 213, 233, 252, 271,
+      291, 311, 332, 357, 382, 406, 428, 449, 468, 485, 500, 511, 517
+    </Vector>
+
+    <Vector name="Offset_Front_Right" type="numeric" nEntries="28">
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5, 6, 7, 8, 8, 7, 6, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+    </Vector>
+
+    <Vector name="Offset_Front_Left" type="numeric" nEntries="28">
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 6, 7, 8, 8, 7, 6, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+    </Vector>
+
+    <Vector name="StartCopyNo_Back_Right" type="numeric" nEntries="28">
+      1, 10, 23, 39, 57, 77, 99, 122, 146, 171, 194, 214, 234, 254, 273,
+      293, 313, 335, 360, 384, 407, 430, 451, 470, 487, 501, 511, 514
+    </Vector>
+
+    <Vector name="StartCopyNo_Back_Left" type="numeric" nEntries="28">
+      1, 4, 14, 28, 45, 64, 85, 107, 130, 154, 179, 201, 221, 241, 260,
+      280, 300, 320, 343, 368, 392, 415, 437, 457, 475, 491, 504, 513
+    </Vector>
+
+    <Vector name="Offset_Back_Right" type="numeric" nEntries="28">
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 6, 7, 7, 8, 7, 6, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+    </Vector>
+
+    <Vector name="Offset_Back_Left" type="numeric" nEntries="28">
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 6, 7, 8, 7, 7, 6, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+    </Vector>
+
+  </ConstantsSection>
+
+  <RotationSection label="etl.xml">
+    <Rotation name="0" thetaX="90*deg" phiX="0*deg" thetaY="90*deg" phiY="90*deg" thetaZ="0*deg" phiZ="0*deg"/>
+    <Rotation name="270" thetaX="90*deg" phiX="270*deg" thetaY="90*deg" phiY="0*deg" thetaZ="0*deg" phiZ="0*deg"/>
+    <Rotation name="180" thetaX="90*deg" phiX="180*deg" thetaY="90*deg" phiY="270*deg" thetaZ="0*deg" phiZ="0*deg"/>
+    <Rotation name="90" thetaX="90*deg" phiX="90*deg" thetaY="90*deg" phiY="180*deg" thetaZ="0*deg" phiZ="0*deg"/>
```

Comment:
Could these rotations please be revised? They look like duplicates of existing rotations. Could you use rotations from rotations.xml rather than duplicating them here?
If any of these rotations have to be retained, they should be given readable, unique names. A name like "0" could cause confusion.

Reply:
Hi Carl, I created a new version of the xml in which I use the rotations from rotations.xml. You can find it here: https://github.com/martatornago/cmssw/commit/179e51344d74ca2236ae1740fa71acd4c58a522a
I checked the geometry is unchanged after changing the rotations using testMTDinDDD, OverlapChecks and Fireworks.

Reply:
@cvuosalo thanks for the remark, @martatornago has updated the code accordingly and verified it is ok